crates/swc_ecma_transforms_base/src/perf.rs

```diff
@@ -50,38 +47,26 @@ where
         N: Send + Sync + VisitWith<Self>,
     {
         if nodes.len() >= threshold || option_env!("SWC_FORCE_CONCURRENT") == Some("1") {
-            GLOBALS.with(|globals| {
-                HELPERS.with(|helpers| {
-                    HANDLER.with(|handler| {
-                        use rayon::prelude::*;
-
-                        let visitor = nodes
-                            .into_par_iter()
-                            .map(|node| {
-                                GLOBALS.set(globals, || {
```

Comment:
the `globals`,  from `GLOBALS.with` they are just the same thing, reset the `GLOBALS` here is also a little redundant